settings.gradle.kts

```diff
@@ -5,13 +5,6 @@ pluginManagement {
         jcenter()
         mavenCentral()
     }
-    resolutionStrategy {
```

Comment:
any reasons removing this block?

Reply:
One reason is for `android gradle` version inconsistency.
Since settings cannot have shared variable for android gradle version in buildSrc.
Another reason is that it seems this script don't change anything 
because in the end gradle use `android gradle` class path specify in the androidApp/build.gradle.kts

But let me know if this removing affect anything unexpectedly

Reply:
let's keep it atm maybe?

Reply:
Sure mas

Reply:
Done mas https://github.com/radityagumay/QualityMatters/pull/1/commits/a46ac830a04969453c234c2f3151bb8081043347

Reply:
thank you!